Guerrero_Melissa/Assignments/multiply_function.py

```diff
@@ -0,0 +1,9 @@
+def multiply(count):
+	x = count*5
+	return x
+
+a = [2, 4, 10, 16]
+for count in a:
+	multiply(count*5)
+	print count*5
+
```

Comment:
I'm not sure this is what was requested. The platform wanted you to define a function that takes in as parameters a list and a value to scale it by, e.g.:

`def multiply(arr, scaler):`

Also the goal of the function is to return a list. The printing can be done outside the function